examples/provider/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    devops-bootcamp = {
+      source = "liatr.io/terraform/devops-bootcamp"
+    }
+  }
+}
+
+provider "devops-bootcamp" {
+  endpoint = "http://localhost:8080"
```

Comment:
should we be setting a hard endpoint here? I thought terraform was supposed to work for any endpoint?

Reply:
The hard endpoint is set up to find the rest api host. Not quite sure what you mean in terms of supposed to work for any endpoint.